tests/communications/test_websocket_actor.py

```diff
@@ -30,8 +30,9 @@ def test_all_registered_websocket_actor():
 def test_websocket_actor_echo():
     msg = websocket.EchoMsg(msg="echo")
     reply = listener.echo(msg)
-    assert isinstance(reply, websocket.EchoMsg)
+    assert isinstance(reply, websocket.EchoResponse)
     assert reply.msg == msg.msg
+    assert reply.job_ids == []
```

Comment:
You need to add tests for cancel task also

Reply:
done